api.js

```diff
@@ -404,6 +409,80 @@ module.exports = async function attachAPI(app, {wss, db}) {
     next()
   })
 
+  // Don't let users who aren't verified (permissionLevel unauthorized)
+  // do anything with the API. TODO: Be able to disable this!
```

Comment:
Would appreciate permissionLevel being a number rather than a string, by the way. e.g.

```
0 unathorized
1 member
2 admin
```

Reply:
Yeah, I agree - but that's an outdated comment. I'm now using a separate `authorized` property (so, `authorized`: true/false).